dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts

```diff
@@ -62,7 +62,7 @@ export default class SharingOverview extends mixins(EvanComponent) {
 
   data: SharedUserInterface[] = [];
 
-  isActiveSearch = false;
+  isActiveSearch = !!window.localStorage['evan-test-mode'];
```

Comment:
What does this accomplish? :)

Reply:
The test cannot do a proper search with the blur event. For that, this is used to always display the search input.